### PR TITLE
Browser+LibWeb: Tweak favicon loading and remove debug spam

### PR DESCRIPTION
**Pull request by SeekingBlues**
Previously in Browser, when we navigate back from a page that has an icon to a page that does not have an icon, the icon does not update and the old icon is displayed because FrameLoader does not set the default favicon when the favicon cannot be loaded. This patch ensures that Browser receives a new icon bitmap every time a load takes place.

### CODE REVIEW

**Comment by GMTA** [comment]
Could you please move the dbgln changes into a separate commit? That will make it easier to review and bisect/revert in the future.

**Comment by SeekingBlues** [comment]
Yes. The commits are now fixed. :)

File: Userland/Libraries/LibWeb/Loader/FrameLoader.cpp [hunk 1]
```diff
@@ -155,6 +161,11 @@ bool FrameLoader::load(const LoadRequest& request, Type type)
     if (type == Type::IFrame)
         return true;
 
+    auto set_default_favicon = [this] {
+        if (auto* page = browsing_context().page())
+            page->client().page_did_change_favicon(*s_default_favicon_bitmap);
+    };
```
Comment (awesomekling) [inline Userland/Libraries/LibWeb/Loader/FrameLoader.cpp:167]: Seems like this should be a member function on `FrameLoader` instead of a lambda :)

**Review by awesomekling — CHANGES_REQUESTED** [review]

File: Userland/Libraries/LibWeb/Loader/FrameLoader.cpp (hunk 1 above)
Comment (SeekingBlues) [reply Userland/Libraries/LibWeb/Loader/FrameLoader.cpp]: Fixed! :^)

**Pull request merged**